core/app/models/spree/stock/estimator.rb

```diff
@@ -1,12 +1,14 @@
 module Spree
   module Stock
     class Estimator
-      attr_reader :order, :currency
+      attr_reader :order, :currency, :shipping_rate_sorter_class, :shipping_rate_selector_class
 
       # @param order [Spree::Order] the order whose shipping rates to estimate
-      def initialize(order)
+      def initialize(order, shipping_rate_sorter_class = Spree::Config[:shipping_rate_sorter_class], shipping_rate_selector_class = Spree::Config[:shipping_rate_selector_class])
```

Comment:
I think there is no point in both passing the class as a parameter, and having it available as an app configuration.

Have a look at how @jordan-brough did it here: https://github.com/mtomov/solidus/pull/1/files
`order.rb#438` line specifically - just call it directly.